src/components/Authorization/components/Login/login.jsx

```diff
@@ -0,0 +1,85 @@
+import { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import Button from "../../../../common/Button/button";
+import Input from "../../../../common/Input/input";
+
+export const Login = (props) => {
+  const navigate = useNavigate();
+  const name = "";
+  const [email, setemail] = useState("");
+  const [password, setPassword] = useState("");
+  const loginUrl = "http://localhost:4000/login";
+
+  const emailData = (data) => {
+    setemail(data);
+  };
+  const passwordData = (data) => {
+    setPassword(data);
+  };
+
+  function login() {}
+
+  async function formData() {
+    var data = {
+      name,
+      email,
+      password,
+    };
+    const requestOptions = {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify(data),
+    };
+    const response = await fetch(loginUrl, requestOptions);
+    fetch(loginUrl, requestOptions)
+      .then((response) => {
+        console.log(response);
```

Comment:
You can remove console.log

Reply:
Or remove this fetch, as its not doing anything

Reply:
Done